_posts/en/2020-07-28-micro-procurment-whats-stopping-us.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title:  "Micro-procurement for digital work - what's stopping us?"
+ref: working-in-the-open-p1
+lang: en
+author: "Rachel Muston, in collaboration with the IT Strategy team, and procurement experts at TBS and ESDC"
+date: "2020-07-28"
+excerpt_separator: <!--more-->
+
+---
+
+The Government of Canada (GC) buys a lot of stuff. In 2016, the total spent by the GC on goods and services was ~$15.9B. From an IT point of view, this spending can including big ticket goods like new data centres and new national applications costing tens of millions of dollars, and also can include low dollar value items like paying someone to build a small prototype or write a position paper.
```

Comment:
I'm wondering if there should not also be a point regarding: this is not "contract splitting", this is key, point in time needs solved through micro-procurement of services (not goods, right?)

Also, not sure if this fits here or in another blog post but we need to drop the "software is a goods" concept. This new way of obtaining should provide automated artifact tracking in the overall IT architecture. (more I write, the less this fits in the scope of this post...)

Reply:
Ah I wondered about that (each sentence on its own line). Fixed. :-)

re: contract splitting...I'm going to address that in part 2... :-) Check out the google doc for more: https://docs.google.com/document/d/1cYXYUT3ffIN1Qf2zdXWDgxB-AhqzFqLLKv_FttOYbig/edit?usp=sharing

That makes me think... should I be clear somewhere that this is a two parter? (perhaps in the title?)

I've removed the reference to 'goods' in the first paragraph.  :-). In general I try to avoid the goods vs. services convo when it comes to GCDevEx because we are treading a very delicate line re: ProServices (mandatory supply arrangement for all IT services under $100K).

Reply:
Yes, totally could put that as part of the intro: 

>This is part 1 of a series of X posts, where we're looking at ABC. In this first post, we'll focus on A.